readme.md

```diff
@@ -167,171 +180,158 @@ console.log(ulChildren);
 
-> Note: these are just some snippets, read the full [MDN DOM documentation](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) for more examples
+```html
+<input type="text" class="search" placeholder="some text here ....">
+```
 
-## You do: Logo hijack (15 min)
+- `.css()`
+[documentation](http://api.jquery.com/css/)
 
-1. Open up www.google.com in Chrome or Firefox, and open up the console.
-- find an image url for the yahoo logo
-- Store the url to the Yahoo logo in a variable.
-- Find the Google logo using JS and store it in a variable.
-- Modify the source of the logo IMG so that it's a Yahoo logo instead.
-- Find the Google search button and store it in a variable.
-- Modify the text of the button so that it says "Yahooo!" instead.
+- `.val()`
+[documentation](http://api.jquery.com/val/)
 
-Bonus: Add a new element between the image and the search textbox, telling the world that "Yahoo is the new Google".
 
-## Creating/Removing DOM Elements (1 min)
+## Adding content
 
-- [document.createElement](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement)
+`.append()`
+- adds newly created element to the end of a parent element, making it the last child
 
-```js
-var newDiv = document.createElement("div")
-newDiv.innerHTML = "Whoa, this is created from JS!"
+```javascript
+$( "ul" ).append( "<li>blue</li>" );
+});
 ```
 
-- [Node.appendChild](https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild)
+`.prepend()`
+- adds newly created element to the start of a parent element, making it the first child
 
-```js
-document.body.appendChild(newDiv)
+```javascript
+$( "ul" ).prepend( "<li>pink</li>" );
+});
 ```
-- Node.removeChild
 
-## Break (5 min)
+####You Do: Document Dive and examples (10 minutes)
 
-## Events (10 min)
-Let's take a look at this [site](http://jessicahische.is/). We can click on one of these themes and the webpage seemingly completely changes!
+For the remaining two methods, find a partner, research and provide an example for each.
 
-What is an event?
-http://eloquentjavascript.net/14_event.html
+- `.prependTo()`
+[documentation](http://api.jquery.com/prependTo/)
+- `.appendTo()`
+[documentation](http://api.jquery.com/appendTo/)
 
-Events are a way we can add behavior depending on something happening. Think back to cookie clicker. When we click the cookie, something happens.
 
-Lets write some html that contains a couple elements we can click on. Create an `index.html` and put in the following contents:
+## Removing content
 
-```html
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Document</title>
-</head>
-<body>
-  <h1>Click Me!</h1>
-  <a href="https://www.google.com">This is a link to google!</a>
-  <script src="script.js"></script>
-</body>
-</html>
+`.remove()``
+- removes element from DOM
+
+```javascript
+$( "#red" ).remove();
```

Comment:
think you should also include `.hide()` and ask the class the differences between the three(`.hide()`, `.remove()`, `.empty()`